app/tour-step-service.js

```diff
@@ -11,9 +13,10 @@ export default function (Tether, $compile, $document, $templateCache, $rootScope
     function createPopup(step, tour) {
         const scope = angular.extend($rootScope.$new(), {
                 tourStep: step,
-                tour: tour
+                tour: tour,
+                template: stepTemplateHTML
             }),
-            popup = $compile($templateCache.get('tour-step-popup.html'))(scope),
+            popup = $compile(stepPopupHTML)(scope),
```

Comment:
These changes do not allow for overriding.

Reply:
Can this currently be overridden?

Reply:
Yes, by adding a template to the template cache with the same key it will override it.